### PR TITLE
Update azure_rm_cdnprofile to include the Standard and Premium Frontdoor options

### PR DESCRIPTION
**Pull request by jartoo**
##### SUMMARY

Adds the ability to use the existing azure_rm_cdnprofile module to create both a Standard_AzureFrontDoor and Premium_AzureFrontDoor.

Begins to work on https://github.com/ansible-collections/azure/issues/1041 which will require additional pull requests and new modules. This does not complete this issue, however. This also bumps the azure-mgmt-cdn version from 11.0.0 to 13.1.0, which is the latest and will be needed for upcoming pull requests to finish the Front Door (Standard and Premium) functionality.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

Module: azure_rm_cdnprofile
Change: allows additional SKU options.

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by jartoo** [comment]
Additional modules being submitted in the future related to this one:
- azure_rm_afdendpoint ( #1589 )
- azure_rm_afdorigingroup ( #1590 )
- azure_rm_afdorigin ( #1591 )
- azure_rm_afdruleset ( #1592 )
- azure_rm_afdrules ( #1604 )
- azure_rm_afdroute ( #1595 )
- Update requirements to include azure-mgmt-frontdoor==1.2.0 (#1607 )
- azure_rm_frontdoorpolicy (for the related WAF policy functionality)
- azure_rm_afdsecuritypolicy
- azure_rm_afdcustomdomains (will be adding this one)

Others I am considering:
- afd secrets?
- managed rulesets?

**Comment by jartoo** [comment]
Was not sure if you wanted all of the commits for each module done separately or not...assumed separate, though there are some cross dependencies with the tests, so they should be progressively accepted and then tested..tried getting feedback via email, but no engagement.  Hope this is an acceptable approach.  Thanks!

File: plugins/modules/azure_rm_cdnprofile.py [hunk 1]
```diff
@@ -129,7 +131,7 @@ def __init__(self):
             ),
             sku=dict(
                 type='str',
-                choices=['standard_verizon', 'premium_verizon', 'custom_verizon', 'standard_akamai', 'standard_chinacdn', 'standard_microsoft']
+                choices=['standard_verizon', 'premium_verizon', 'custom_verizon', 'standard_akamai', 'standard_chinacdn', 'standard_microsoft', 'standard_azurefrontdoor', 'premium_azurefrontdoor']
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_cdnprofile.py]: line too long (196 > 160 characters)

**Comment by Fred-sun** [comment]
@jartoo small change request! Thank you very much!

File: plugins/modules/azure_rm_cdnprofile.py (hunk 1 above)
Comment (jartoo) [reply plugins/modules/azure_rm_cdnprofile.py]: Appreciate the feedback!  Changed.  Curious if I missed where these type of requirements/coding standards are located?  Happy to conform!

File: plugins/modules/azure_rm_cdnprofile.py [hunk 2]
```diff
@@ -129,7 +131,11 @@ def __init__(self):
             ),
             sku=dict(
                 type='str',
-                choices=['standard_verizon', 'premium_verizon', 'custom_verizon', 'standard_akamai', 'standard_chinacdn', 'standard_microsoft']
+                choices=[
+                    'standard_verizon', 'premium_verizon', 'custom_verizon', 'standard_akamai',
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_cdnprofile.py]: trailing whitespace
```suggestion
                    'standard_verizon', 'premium_verizon', 'custom_verizon', 'standard_akamai',
```

File: plugins/modules/azure_rm_cdnprofile.py [hunk 3]
```diff
@@ -129,7 +131,11 @@ def __init__(self):
             ),
             sku=dict(
                 type='str',
-                choices=['standard_verizon', 'premium_verizon', 'custom_verizon', 'standard_akamai', 'standard_chinacdn', 'standard_microsoft']
+                choices=[
+                    'standard_verizon', 'premium_verizon', 'custom_verizon', 'standard_akamai', 
+                    'standard_chinacdn', 'standard_microsoft',
```
Comment (Fred-sun) [inline plugins/modules/azure_rm_cdnprofile.py]: trailing whitespace
```suggestion
                    'standard_chinacdn', 'standard_microsoft',
```

**Comment by Fred-sun** [comment]
@jartoo small change request! Thank you!

**Comment by Fred-sun** [comment]
@jartoo Please!

**Comment by jartoo** [comment]
Thanks for all your help!  I learned very much with these pull requests.

**Comment by jartoo** [comment]
I updated the cdn library to be 13.1.1 due to this issue, which affects the ability to manage Azure Front Doors (standard and premium)

https://github.com/Azure/azure-sdk-for-python/issues/35801

Otherwise you cannot clear the cache_configuration in Route() and Rule().

**Review by xuzhang3 — APPROVED** [review]

**Pull request merged**